arch/xtensa/src/esp32/Make.defs

```diff
@@ -41,6 +41,11 @@ CMN_CSRCS += xtensa_reprioritizertr.c xtensa_schedsigaction.c
 CMN_CSRCS += xtensa_sigdeliver.c xtensa_stackframe.c xtensa_udelay.c
 CMN_CSRCS += xtensa_unblocktask.c xtensa_usestack.c xtensa_swint.c xtensa_switchcontext.c
 
+
+ifeq ($(CONFIG_DEBUG_TCBINFO),y)
+CMN_CSRCS += xtensa_tcbinfo.c
+endif
+
 # Configuration-dependent common XTENSA files
```

Comment:
```suggestion
# Configuration-dependent common XTENSA files

ifeq ($(CONFIG_DEBUG_TCBINFO),y)
  CMN_CSRCS += xtensa_tcbinfo.c
endif
```

It should be placed below the comment for `Configuration-dependent common XTENSA files`